src/test/SDL_test_harness.c

```diff
@@ -225,10 +225,11 @@ static SDL_TimerID SDLTest_SetTestTimeout(int timeout, void(SDLCALL *callback)(v
 #ifdef __WATCOMC__
 #pragma aux SDLTest_BailOut aborts;
 #endif
-static SDL_NORETURN void SDLCALL SDLTest_BailOut(void)
+static Uint32 SDLCALL SDLTest_BailOut(void *userdata, SDL_TimerID timerID, Uint32 interval)
 {
     SDLTest_LogError("TestCaseTimeout timer expired. Aborting test run.");
     exit(TEST_ABORTED); /* bail out from the test */
+    return 0;
```

Comment:
`return TEST_ABORTED;` for consistency?

Reply:
> `return TEST_ABORTED;` for consistency?

`SDL_TimerCallback` returns the new time interval. 0 disables it. `TEST_ABORTED` is `-1` and would return the max `Uint32` value.

Reply:
OK